.gitignore

```diff
@@ -1,2 +1,3 @@
 /target*
 /rust.git
+/*.dot
```

Comment:
Is this needed? Whats writing dot files to project root?

Reply:
Im emitting them from the rga example to cwd, which is typically the root of the repo.

Reply:
It's also handy to pipe stdout to a dot file and then visualize the result; again the dot file lands at the top level. I suppose we could ignore dot files at every level..

Reply:
Ok sounds good to me to keep as-is